src/main/kotlin/StringCalculator.kt

```diff
@@ -0,0 +1,12 @@
+class StringCalculator {
```

Comment:
How do you see the difference between a class with state (fields) and a stateless class?
What are your thoughts on the difference between stateful and stateless classes?